pmm-app/docker-compose.yaml

```diff
@@ -24,19 +30,19 @@ services:
 
   pmm-client:
     container_name: pmm-client
-    image: centos
+    image: rockylinux:9
```

Comment:
I suggest we use oraclelinux:9 instead, which is the OS we support and use across the board, including the CI environment.

Reply:
Good point, updated.